code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -712,3 +721,38 @@
 	nutriment_factor = 5 * REAGENTS_METABOLISM
 	color = "#eef442" // rgb: 238, 244, 66
 	taste_description = "mournful honking"
+
+/datum/reagent/consumable/ghostchilijuice
+	name = "Ghost Chili Juice"
+	id = "ghostchiilijuice"
+	description = "A greish juice with the same properties of lava."
+	color = "#bfaf90" //rgb 191, 175, 144
+	nutriment_factor = 0.0001 * REAGENTS_METABOLISM
+	taste_description = "lava"
+
+/datum/reagent/consumable/ghostchilijuice/on_mob_life(mob/living/carbon/M)
+
+	if(prob(5))
+		M.adjust_fire_stacks(2)
+		M.IgniteMob()
+		M.say("Spicy!")
+	M.adjust_bodytemperature(50 * TEMPERATURE_DAMAGE_COEFFICIENT)
+	..()
+
+/datum/reagent/consumable/ghostchilijuice/reaction_mob(mob/living/M, method=TOUCH, reac_volume)
+	M.adjustFireLoss(reac_volume/5, 0)
+	if(!ishuman(M) && !ismonkey(M))
+		return
+	if(!M.is_eyes_covered())
+		M.emote("scream")
```

Comment:
Could we add some sort of cooldown to the scream emote? Otherwise it will be spammed like crazy when used in smoke. 

Liking the actual chem though.